### ISSUE TITLE
Benchmark fails to build on Stackage nightly

### ISSUE THREAD

**Issue by mihaimaruseac** [rudymatela/fitspec#1]
```haskell
Preprocessing benchmark 'mergeheaps' for fitspec-0.4.2..
Building benchmark 'mergeheaps' for fitspec-0.4.2..

<no location info>: warning: [-Wmissing-home-modules]
    These modules are needed for compilation but not listed in your .cabal file's other-modules: Test.FitSpec
                                                                                                 Test.FitSpec.Derive
                                                                                                 Test.FitSpec.Engine
                                                                                                 Test.FitSpec.Main
                                                                                                 Test.FitSpec.Mutable
                                                                                                 Test.FitSpec.Mutable.Tuples
                                                                                                 Test.FitSpec.PrettyPrint
                                                                                                 Test.FitSpec.Report
                                                                                                 Test.FitSpec.ShowMutable
                                                                                                 Test.FitSpec.ShowMutable.Tuples
                                                                                                 Test.FitSpec.TestTypes
                                                                                                 Test.FitSpec.Utils
[ 1 of 13] Compiling Test.FitSpec.Mutable ( src/Test/FitSpec/Mutable.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Mutable.o )
[ 2 of 13] Compiling Test.FitSpec.Mutable.Tuples ( src/Test/FitSpec/Mutable/Tuples.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Mutable/Tuples.o )
[ 3 of 13] Compiling Test.FitSpec.PrettyPrint ( src/Test/FitSpec/PrettyPrint.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/PrettyPrint.o )
[ 4 of 13] Compiling Test.FitSpec.ShowMutable ( src/Test/FitSpec/ShowMutable.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/ShowMutable.o )
[ 5 of 13] Compiling Test.FitSpec.Derive ( src/Test/FitSpec/Derive.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Derive.o )
[ 6 of 13] Compiling Test.FitSpec.ShowMutable.Tuples ( src/Test/FitSpec/ShowMutable/Tuples.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/ShowMutable/Tuples.o )
[ 7 of 13] Compiling Test.FitSpec.TestTypes ( src/Test/FitSpec/TestTypes.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/TestTypes.o )
[ 8 of 13] Compiling Test.FitSpec.Utils ( src/Test/FitSpec/Utils.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Utils.o )
[ 9 of 13] Compiling Test.FitSpec.Engine ( src/Test/FitSpec/Engine.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Engine.o )
[10 of 13] Compiling Test.FitSpec.Report ( src/Test/FitSpec/Report.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Report.o )
[11 of 13] Compiling Test.FitSpec.Main ( src/Test/FitSpec/Main.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec/Main.o )
[12 of 13] Compiling Test.FitSpec     ( src/Test/FitSpec.hs, dist/build/mergeheaps/mergeheaps-tmp/Test/FitSpec.o )
[13 of 13] Compiling Main             ( bench/mergeheaps.hs, dist/build/mergeheaps/mergeheaps-tmp/Main.o )

bench/mergeheaps.hs:7:1: error:
    Could not find module ‘Heap’
    Use -v to see a list of the files searched for.
  |
7 | import Heap
  | ^^^^^^^^^^^
```

**Comment by rudymatela** [comment]
I'm sorry about the failure.  Thanks for reporting the issue.

I'll look into this later today.

**Comment by rudymatela** [comment]
Turns out the `other-modules` field for some benchmarks on cabal were missing.  Commit 1c65d60 fixes this.  I have added a test (ca0c4eb) so that I do not make this mistake again.

I have released [FitSpec v0.4.3] which should not have this problem.  [FitSpec v0.4.3 builds and tests fine on Travis](https://travis-ci.org/rudymatela/fitspec/builds/263137244).

When you have the time, you can revert fpco/stackage@4a29c8e too see if it works.

Again, thanks for the bug report.

[FitSpec v0.4.3]: https://hackage.haskell.org/package/fitspec-0.4.3

**Comment by mihaimaruseac** [comment]
Thank you. It builds properly now.
